cloudbuild/presubmit.sh

```diff
@@ -38,7 +38,7 @@ initialize_git_repo() {
 }
 
 # Determines whether a given string is a prefix string of any changed file name
-is_prefix() {
+is_changed_prefix() {
```

Comment:
Instead of enumerating all directories and see if they include changed files, can we find these directories directly from changed files? I think in this way, we can avoid recursion.

Reply:
Done.